### ISSUE TITLE
Issue when using Rack Server with Rails 3

### ISSUE THREAD

**Issue by maxschulze** [demandforce/rack-oauth2-server#16]
When use the initializer for Rails3 like this, I get an error complaining about the mongo database not beeing set.

```
# configure rack-oauth2-server
Frontdesk::Application.configure do
  config.after_initialize do
    # integrate with devise
    config.oauth.database = Mongo::Connection.new()['test_db']
    config.oauth.authenticator = lambda do |email, password|
      user = User.find_for_database_authentication(:email => email)
      user if user && user.valid_password?(password)
    end
  end
end
```

I fixed it by putting 

```
Rack::OAuth2::Server.database = Mongo::Connection.new()['test_db']
```

after the initializer as well.

**Comment by radicaled** [comment]
Having the same problem.

Looks like it started breaking with this pull, I think: https://github.com/flowtown/rack-oauth2-server/pull/10

**Comment by maxschulze** [comment]
Hum, but why this seems to be just an accessor of @database?

**Comment by assaf** [comment]
Fixed in 2.3.0.

**Comment by maxschulze** [comment]
Thanks!
